x-pack/performance/journeys/saved_object_dashboard_list.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Journey } from '@kbn/journeys';
+
+export const journey = new Journey({
+  esArchives: ['x-pack/performance/es_archives/sample_data_flights'],
+  kbnArchives: [
+    'x-pack/performance/kbn_archives/flights_no_map_dashboard',
+    'x-pack/performance/kbn_archives/logs_no_map_dashboard',
+  ],
+}).step('Go to Dashboards Page', async ({ page, kbnUrl }) => {
+  await page.goto(kbnUrl.get(`/app/dashboards`));
+  await page.waitForSelector('#dashboardListingHeading');
```

Comment:
need to double check if this is actually waiting for display of the items.

Reply:
I can see the events in telemetry cluster, so journey is probably does not require any extra waiting.

<img width="2032" alt="Screenshot 2022-12-14 at 17 23 15" src="https://user-images.githubusercontent.com/10977896/207651598-165c6d72-45a8-4e2f-897d-46a6c7665734.png">

I wonder if we really need a new journey to get the metric? We can get it from the existing journeys already
<img width="1685" alt="image" src="https://user-images.githubusercontent.com/10977896/207652889-3dbf8a6e-1dfa-4d97-8d9f-bc018d8ce33b.png">
Maybe it makes sense to add a journey with many saved objects (100+) and then we can compare it with sample data?

Reply:
> I can see the events in telemetry cluster, so journey is probably does not require any extra waiting.

Looking into it, waiting for the header is not the right event to end the journey. While it works for a first page load, it will not when we write journeys where users type in custom searches.

I will introduce a new event that allows to check until the data has been put on screen.

> I wonder if we really need a new journey to get the metric?

Makes sense. I just like wanted to get some boilerplate to isolate the dashboard page, since we will be adding journeys for deletion as well.

> Maybe it makes sense to add a journey with many saved objects (100+) and then we can compare it with sample data?

Yes, but will do this in separate PR.